Corona-Warn-App/src/main/res/layout/include_risk_card_content.xml

```diff
@@ -1,281 +1,287 @@
-                app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintEnd_toStartOf="@+id/risk_card_header_icon"
+                android:layout_marginEnd="16dp"
+                android:accessibilityHeading="true"
+                android:text="@{tracingCard.getProgressCardHeadline(context)}"
+                android:textColor="@color/colorStableLight"
+                app:layout_constraintEnd_toStartOf="@+id/risk_card_progress_headline_icon"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintTop_toTopOf="parent"
+                app:layout_goneMarginEnd="0dp"
+                tools:text="Daten werden herunter geladen" />
 
             <ImageView
-                android:id="@+id/risk_card_header_icon"
+                android:id="@+id/risk_card_progress_headline_icon"
+                gone="@{tracingCard.showDetails}"
                 android:layout_width="@dimen/icon_size_risk_card"
                 android:layout_height="@dimen/icon_size_risk_card"
                 android:importantForAccessibility="no"
                 android:src="@drawable/ic_forward"
-                android:tint="@{tracingCard.getStableTextColor(context)}"
-                gone="@{tracingCard.showDetails}"
+                android:tint="@color/colorStableLight"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintTop_toTopOf="parent" />
 
-        </androidx.constraintlayout.widget.ConstraintLayout>
-
-        <!-- Loading State -->
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="@dimen/match_constraint"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/spacing_normal"
-            gone="@{!tracingCard.refreshing}"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/risk_card_header">
-
             <ProgressBar
-                android:id="@+id/risk_card_progress"
+                android:id="@+id/risk_card_progress_indicator"
                 style="@style/progressBar"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_width="36dp"
+                android:layout_height="36dp"
                 android:indeterminate="true"
-                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintBottom_toBottomOf="@+id/textView2"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintTop_toTopOf="@+id/textView2" />
 
             <TextView
+                android:id="@+id/textView2"
```

Comment:
use a better id instead of `textView2`, to improve readability and match the other ids.